src/components/Card/Card.js

```diff
@@ -1,8 +1,45 @@
+import './Card.css'
 import createElement from '../../lib/createElement'
 
-export default function Card() {
-  const el = createElement('section', {
-    className: 'Card',
-  })
-  return el
+export default function Card(charData, charImage) {
+  return createElement(
+    'section',
+    {
+      className: 'Card',
+    },
+    createImage(charImage),
+    createList(charData)
+  )
+
+  function createImage(charImage) {
+    const imgURL = charImage
+    const imgEl = createElement('img', {
+      className: 'Card__image',
+      src: imgURL,
+      width: 200,
+    })
+    const imgFrame = createElement(
+      'figure',
+      { className: 'Card__frame' },
+      imgEl
+    )
+    return imgFrame
+  }
+
+  function createList(charData) {
+    const listItems = charData
+      .map(data => {
+        if (data[1]) {
+          return `<dt class="Card_label">${data[0]}:</dt><dd class="Card_value">${data[1]}</dd>`
+        } else {
+          return `<dt class="Card_label">${data[0]}:</dt><dd class="Card_value">n.A.</dd>`
+        }
+      })
+      .join('')
+
+    const cardList = createElement('dd', { className: 'Card__list' })
+    cardList.innerHTML = listItems
+
+    return cardList
+  }
```

Comment:
With my suggestions above:

```suggestion
  function createList(charData) { // charData would be an object here (like suggested above)
    const listItems = Object.entries(charData) // creates a nested array like you did by yourself
      .map(([name, value]) =>  // destructuring assignment of an array
        `<dt class="Card_label">${name}:</dt><dd class="Card_value">${value ?? 'n.A.'}</dd>`
      )
    return createElement('dd', { className: 'Card__list' }, ...listItems)
  }
```